core/arch/riscv/mm/core_mmu_arch.c

```diff
@@ -246,7 +247,7 @@ static struct mmu_partition *core_mmu_get_prtn(void)
 
 static struct mmu_pgt *core_mmu_get_root_pgt_va(struct mmu_partition *prtn)
 {
-	return prtn->root_pgt;
+	return &prtn->root_pgt[get_core_pos()];
```

Comment:
suggestion `	return prtn->root_pgt + get_core_pos();`

Reply:
Changed in https://github.com/OP-TEE/optee_os/pull/6713/commits/79246e385fd59a63047ac7d201c1d8ab75196c31